### ISSUE TITLE
Error after removing breaks 

### ISSUE THREAD

**Issue by System25** [slgobinath/SafeEyes#310]
Hi, I removed the "rolling eyes" breaks (hourly, and side to side), and after that safeeyes doesn't work any more.

If I run safeeyes in command line the following errors appears:

```
  File "/usr/lib/python3/dist-packages/safeeyes/Utility.py", line 93, in <lambda>
    GLib.idle_add(lambda: target_function(args))
  File "/usr/lib/python3/dist-packages/safeeyes/SafeEyesCore.py", line 217, in __fire_on_update_next_break
    self.on_update_next_break.fire(self.break_queue.get_break(), next_break_time)
  File "/usr/lib/python3/dist-packages/safeeyes/model.py", line 225, in fire
    if not handler(*args, **keywargs):
  File "/usr/lib/python3/dist-packages/safeeyes/SafeEyes.py", line 257, in update_next_break
    self.plugins_manager.update_next_break(break_obj, break_time)
  File "/usr/lib/python3/dist-packages/safeeyes/PluginManager.py", line 163, in update_next_break
    plugin['module'].update_next_break(break_obj, break_time)
  File "/usr/lib/python3/dist-packages/safeeyes/plugins/trayicon/plugin.py", line 381, in update_next_break
    tray_icon.next_break_time(next_break_time)
  File "/usr/lib/python3/dist-packages/safeeyes/plugins/trayicon/plugin.py", line 247, in next_break_time
    self.__set_next_break_info()
  File "/usr/lib/python3/dist-packages/safeeyes/plugins/trayicon/plugin.py", line 254, in __set_next_break_info
    message = _('Next break at %s') % (formatted_time)
TypeError: not all arguments converted during string formatting

(safeeyes:17467): Gdk-CRITICAL **: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/safeeyes/Utility.py", line 93, in <lambda>
    GLib.idle_add(lambda: target_function(args))
  File "/usr/lib/python3/dist-packages/safeeyes/SafeEyesCore.py", line 217, in __fire_on_update_next_break
    self.on_update_next_break.fire(self.break_queue.get_break(), next_break_time)
  File "/usr/lib/python3/dist-packages/safeeyes/model.py", line 225, in fire
    if not handler(*args, **keywargs):
  File "/usr/lib/python3/dist-packages/safeeyes/SafeEyes.py", line 257, in update_next_break
    self.plugins_manager.update_next_break(break_obj, break_time)
  File "/usr/lib/python3/dist-packages/safeeyes/PluginManager.py", line 163, in update_next_break
    plugin['module'].update_next_break(break_obj, break_time)
  File "/usr/lib/python3/dist-packages/safeeyes/plugins/trayicon/plugin.py", line 381, in update_next_break
    tray_icon.next_break_time(next_break_time)
  File "/usr/lib/python3/dist-packages/safeeyes/plugins/trayicon/plugin.py", line 247, in next_break_time
    self.__set_next_break_info()
  File "/usr/lib/python3/dist-packages/safeeyes/plugins/trayicon/plugin.py", line 254, in __set_next_break_info
    message = _('Next break at %s') % (formatted_time)
TypeError: not all arguments converted during string formatting
Exception in thread WorkThread:
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3/dist-packages/safeeyes/plugins/smartpause/plugin.py", line 132, in __start_idle_monitor
    next_break = next_break_time + idle_period
TypeError: unsupported operand type(s) for +: 'NoneType' and 'datetime.timedelta'
```

Best regards!

**Comment by slgobinath** [comment]
Hi,

Did you remove from the UI (Settings -> Breaks dialog)? Could you please share the output of `locale` command?

**Comment by System25** [comment]
Hi, yes I removed it from the UI (Settings -> Breaks dialog).

```
amacias@F0602HWPC0079:~$ locale
LANG=es_ES.UTF-8
LANGUAGE=es_ES
LC_CTYPE="es_ES.UTF-8"
LC_NUMERIC="es_ES.UTF-8"
LC_TIME="es_ES.UTF-8"
LC_COLLATE="es_ES.UTF-8"
LC_MONETARY="es_ES.UTF-8"
LC_MESSAGES="es_ES.UTF-8"
LC_PAPER="es_ES.UTF-8"
LC_NAME="es_ES.UTF-8"
LC_ADDRESS="es_ES.UTF-8"
LC_TELEPHONE="es_ES.UTF-8"
LC_MEASUREMENT="es_ES.UTF-8"
LC_IDENTIFICATION="es_ES.UTF-8"
LC_ALL=
amacias@F0602HWPC0079:~$ 
```
Thanks

**Comment by slgobinath** [comment]
Hi,
This is a bug appears only for Spanish locale. Please apply the fix given in the last comment of #306
